modules/widget_utils.js

```diff
@@ -153,12 +156,19 @@ exports.swipeOn = function(autohide) {
     let cb;
     if (exports.autohide > 0) cb = function() {
       exports.hideTimeout = setTimeout(function() {
+        Bangle.emit("widgets-start-hide");
```

Comment:
Would it work if we moved the `widgets-start-show/hide` into the start of the `anim()` function?